### DISCUSSION TITLE
Any tips to optimise publish latency

### DISCUSSION THREAD

**Discussion by namreg** [centrifugal/centrifugo#771] [Q&A]
Hey,

We use centrifugo v2.8.0 in production. Currently, our publish latency is ~200ms. Seems like it is huge.
Centrifugo deployed in K8S cluster using redis engine (also deployed in the cluster). Centrifugo deployment has enough resources.
```
    Limits:
      cpu:     200m
      memory:  256Mi
    Requests:
      cpu:      50m
      memory:   64Mi
```
Any tips how can we decrease the latency? 
Thanks.
![CleanShot 2024-02-04 at 15 47 42@2x](https://github.com/centrifugal/centrifugo/assets/391872/8d85db8b-46fa-43d9-9cf2-1f4d8bd68beb)
![CleanShot 2024-02-04 at 15 53 08@2x](https://github.com/centrifugal/centrifugo/assets/391872/4eb41104-c455-4cd2-82db-35ab3fb9bef2)

**Comment by FZambia** [comment]
Hello, yep - it seems huge, usually it's microseconds time. Also, v2.8.0 is outdated.

I can only suggest reproducing this on the local machine using latest Centrifugo version and provide the reproducer. Without doing this it's impossible to help – there are plenty of factors which could affect latency including message payload size, features enabled.
